src/ApparentHorizons/Tags.hpp

```diff
@@ -344,6 +344,28 @@ struct UnitNormalVectorCompute : UnitNormalVector<Frame>, db::ComputeTag {
   using return_type = tnsr::I<DataVector, 3, Frame>;
 };
 
+///SpatialRicciScalar is the two-dimensional intrinsic Ricci scalar curvature
```

Comment:
Need a space after `///`    Also this is not the spatial Ricci scalar, so it shouldn't be called that...  isn't `StrahlkorperTags::RicciScalar`  sufficient?

Reply:
Yes, that should be sufficient, I'll make the changes now.